package io.openfuture.chain.exception

class ValidationException(message: String?) : RuntimeException(message)